src/inausoft.netCLI.Tests/LogicalCommandDeserializerTests.cs

```diff
@@ -92,7 +136,23 @@ public void RegexArgumentHandler_ThrowsDeserializationException_ForInvalidOption
         }
 
         [TestMethod]
-        public void RegexOptionsDeserializer_Deserialize_ThrowsDeserializationException_WhenNoValueWasSpecifiedForNotBooleanProperty()
+        public void LogicalCommandDeserializer_ThrowsDeserializationException_WhenOptionIsSuppliedWithOneHyphen()
```

Comment:
If we want to check only if DeserializationException was thrown, maybe a good solution is to put parameter to test method and remove assert here. Name suggest we want to catch DeserializationException but in Assert we check ErrorCode, to discuss :)
https://stackoverflow.com/questions/933613/how-do-i-use-assert-to-verify-that-an-exception-has-been-thrown

Reply:
Note that we both want to see if the exception was thrown and if the associated code is correct. I've renamed test methods to reflect that.

Reply:
Fine for me.